Nodejs/Product/Nodejs/Project/NodeModulesNode.cs

```diff
@@ -362,9 +377,9 @@ private class NpmPathProvider : INpmPathProvider {
             }
         }
 
-#endregion
+        #endregion
 
-#region Updating module hierarchy
+        #region Updating module hierarchy
```

Comment:
Yes, I don't recall touching these lines directly in the first place so I think they were auto formatted. I can revert the change, but this seems to be what other places are doing.